draft-ietf-quic-version-negotiation.md

```diff
@@ -194,92 +215,49 @@ server.
 During the handshake, endpoints will exchange handshake version information,
 which is a blob of data that is defined below. In QUIC version 1, the handshake
 version information is transmitted using a new transport parameter,
-`version_negotiation`. The contents of handshake version information depend on
-whether the client or server is sending it, and are shown below (using the
-notation from the "Notational Conventions" section of {{QUIC}}):
+`version_negotiation`. The contents of handshake version information are shown
+below (using the notation from the "Notational Conventions" section of
+{{QUIC}}):
 
 ~~~
-Client Handshake Version Information {
+Handshake Version Information {
   Currently Attempted Version (32),
```

Comment:
I think that this might be better named "Chosen Version".  It's not necessarily the first choice, but the choice that is made after seeing what the other side supports (the client chooses by guessing if it doesn't get VN, and chooses from VN if it does; the server chooses from compatible versions).

Reply:
Agreed, done.